tests/ftests/systemd.py

```diff
@@ -70,7 +71,14 @@ def remove_scope_slice_conf(config, _slice, scope, controller, config_fname=None
         # In case the error occurs before the creation of slice/scope and
         # we may very well be on the teardown path, ignore the exception
         try:
-            Cgroup.delete(config, controller, cgname=_slice, ignore_systemd=True)
+            CgroupCli.delete(config, controller, cgname=_slice, ignore_systemd=True)
         except RunError as re:
             if 'No such file or directory' not in re.stderr:
                 raise re
+
+    # Check if libcgroup is compiled with --enable-systemd
+    @staticmethod
+    def is_systemd_enabled(config):
```

Comment:
nitpick - `config` isn't being used.  Do you have a plan for it in a later commit?

Reply:
@drakenclimber Thank you so much for reviewing.  `config` is not required, will remove it.